RadixWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -185,8 +185,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/radixdlt/swift-composable-architecture",
       "state" : {
-        "branch" : "full-scope-1.4.2",
-        "revision" : "789fab0d836fb230b5e279f4a62d9112e4d6d81e"
+        "revision" : "30ab89daa4e392ecd2150f737c42dbb2336bbc04",
+        "version" : "1.7.0"
```

Comment:
Not resolved to latest 1.7.3?

Reply:
Actually not so strange, it was still using our fork, but the main branch there. Now I switched to the official repo.